src/ipc/ipc4/dai.c

```diff
@@ -125,6 +124,8 @@ int ipc_dai_data_config(struct comp_dev *dev)
 		return -EINVAL;
 	}
 
+	dai->dma_buffer_size = copier_cfg->gtw_cfg.dma_buffer_size;
+
```

Comment:
This seems completely wrong to me. The deep-buffering is located before the mixer, there is no way a DAI could be impacted by a change in the DMA host buffer size.